config/routes/overrides.rb

```diff
@@ -1,6 +1,21 @@
+class WhitelistConstraint
+  def initialize
+    @path = /^\/api\/v2\/organizations\/\S+\/parameters/
```

Comment:
looks like a regex, so i think its fine as is.   I could maybe see it defined as a constant instead of an instance variable though